test/everypolitician_test.rb

```diff
@@ -100,7 +100,7 @@ def test_alternative_constant_name
   end
 
   def test_retrieving_popolo
-    VCR.use_cassette('popolo') do
+    VCR.use_cassette('popolo', :record => :new_episodes) do
```

Comment:
I was a little surprised that rubocop didn't complain about you using the old-fashioned hash syntax here (rather than `record: :new_episodes`), but that'll be because we don't actually _have_ rubocop running here yet! 

I'll add that today, which will auto-fix this, so no need to change it here, but worth noting for future reference…